app/assets/javascripts/ember/controllers/search.coffee

```diff
@@ -76,9 +76,9 @@ App.SearchController = Ember.Controller.extend
     viewPortTop = @get('viewPortStartRow') * @get('rowHeight')
     viewPortSize = @get('viewPortRowCount') * @get('rowHeight')
     scrollTop = @get('window.scrollTop')
-    if scrollTop < viewPortTop || scrollTop > viewPortTop + viewPortSize - @get('window.height')
-      console.log "redraw"
-      @set 'scrollTop', scrollTop
+    #if scrollTop < viewPortTop || scrollTop > viewPortTop + viewPortSize - @get('window.height')
```

Comment:
Let's leave this in, there's no sense in asking for a redraw with every single scroll event.